src/server.h

```diff
@@ -1515,6 +1523,9 @@ struct redisServer {
     char *proc_title_template;      /* Process title template format */
     clientBufferLimitsConfig client_obuf_limits[CLIENT_TYPE_OBUF_COUNT];
     int pause_cron;                 /* Don't run cron tasks (debug) */
+    int latency_track_enabled;      /* 1 if extended latency tracking is enabled, 0 otherwise. */
+    double *latency_track_percentiles; /* Extended latency tracking percentile list */
+    int latency_percentiles_len;
```

Comment:
let's give this one the same prefix of the config it controls (with a `_len` suffix).
i vote for `latency_tracking_percentiles_len`.

p.s. maybe the comment should state that this is a config.

Reply:
>  i vote for latency_tracking_percentiles_len.

moved to `latency_tracking_info_percentiles` and `latency_tracking_info_percentiles_len` given the discussion about the info topic.